proto/heimdall/types/divident.proto

```diff
@@ -14,8 +14,7 @@ message DividendAccount {
     option (gogoproto.goproto_getters)  = false;
     option (gogoproto.goproto_stringer) = false;
 
-    string user = 1
-        [(gogoproto.casttype) =
-        "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+    string user       = 1 [(gogoproto.casttype) =
```

Comment:
remove casttype to align with other definitions

Reply:
This has been addressed in gov_porting PR.

Reply:
Addressed in another PR (Gov)